roles/sidekiq_worker/.ansible-lint

```diff
@@ -1,3 +1,4 @@
 skip_list:
   - '303'
   - '306'
+  - '403'
```

Comment:
I'd feel more comfortable with this if each item had a comment describing why we're skipping the rule. Otherwise I have the uncomfortable feeling we're sweeping things under the rug with no easy way to revisit, lift up the rug, and clean underneath it.